app/assets/javascripts/components/activity/activity_table.jsx

```diff
@@ -121,7 +108,7 @@ const ActivityTable = createReactClass({
   _renderHeaders() {
     return this.props.headers.map((header) => {
       return (
-        <th style={header.style || {}} key={header.key} onClick={this.sortItems} className="sortable" data-sort-key={header.key}>
+        <th style={header.style || {}} key={header.key} onClick={this.sortItems} className="sortable asc" data-sort-key={header.key}>
```

Comment:
Ideally, this should be done the same way as the `/explore` table: the class changes according to the direction it's being sorted, so the sorting indicator shows which way that column is being sorted. I'll open a separate issue for this.

Reply:
@ragesoss Okay, I understand. I can also work on that issue if you want